docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -11,7 +11,7 @@ Templates are defined in template descriptor files as a JSON array:
 ```json
 [
   {
-    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.2.0/resources/schema.json",
+    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.9.1/resources/schema.json",
```

Comment:
We could also consider just using _latest_ in any snippet because it shouldn't matter. We inform about the latest informed versions already in the docs themselves. 

```suggestion
    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema/resources/schema.json",
```